examples/async-sqlite/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "async-sqlite"
+version = "0.1.0"
+description = "Neon Async SQLite"
+authors = ["K.J. Valencik"]
```

Comment:
We should also probably remove this to be consistent with other examples

Reply:
Done